README.md

```diff
@@ -11,6 +15,7 @@ Optional:
 - Kubernetes
 - Helm
 
+## Configuration steps
 ### Azure Service Bus
 
 This service depends on a valid Azure Service Bus connection string for
```

Comment:
Para below this is missing same ending as `batch-processor`:

When deployed into an appropriately configured AKS cluster (where [AAD Pod Identity](https://github.com/Azure/aad-pod-identity) is configured) the microservice will use AAD Pod Identity through the manifests for [azure-identity](https://github.com/DEFRA/ffc-pay-batch-processor/blob/sfi-2004-update-readme/helm/ffc-pay-batch-processor/templates/azure-identity.yaml) and [azure-identity-binding](https://github.com/DEFRA/ffc-pay-batch-processor/blob/sfi-2004-update-readme/helm/ffc-pay-batch-processor/templates/azure-identity-binding.yaml).